### ISSUE TITLE
motion sequence video upload error?

### ISSUE THREAD

**Issue by frankchieng** [magic-research/magic-animate#26]
after i tinker with detectron2 densepose, i generate motion detection like this

https://github.com/magic-research/magic-animate/assets/130369523/714f47c6-71fd-449d-88e6-3ed7abf1c716

what's the next step if i wanna to upload motion sequence video successfully?

**Comment by FurkanGozukara** [comment]
how did you managed to make DensePose
I cant make it generate : https://github.com/facebookresearch/detectron2/issues/5170

**Comment by FurkanGozukara** [comment]
Full scripts released including DensePose maker : https://github.com/magic-research/magic-animate/issues/44

**Comment by frankchieng** [comment]
problem solved,just uncomment size=int(size) in function read_video of demo/gradio_animate.py

**Comment by zcxu-eric** [comment]
> problem solved,just uncomment size=int(size) in function read_video of demo/gradio_animate.py

Yes, this is a bug in the initial version, it has been fixed. Thanks for your interest!
